src/common/StatusBadge/LeavesStatusBadge.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Badge } from "@/components/ui/badge";
+import { LeavesRequestStatus } from "@/types";
+
+const statusClasses = {
+  accepted: {
+    background: "bg-status-active-background",
+    text: "text-status-active",
+  },
+  pending: {
+    background: "bg-status-pending-background",
+    text: "text-status-pending",
+  },
+  rejected: {
+    background: "bg-status-inactive-background",
+    text: "text-status-inactive",
+  },
+} as const;
+
+export const LeavesStatusBadge = ({
```

Comment:
@nadirhussain786 we can improve ```ts type LeavesRequestStatus``` assign to `status` i have already mentioned

Reply:
Fixed